openpype/lib/avalon_context.py

```diff
@@ -2089,3 +2090,97 @@ def _filter_input_links(input_links, link_type, correctly_linked_ids):
                                      input_link.get("_id"))  # legacy
 
     return correctly_linked_ids
+
+
+def get_asset_hierarchy_tree(project_name):
+    """Returns list of nodes mapping hierarchy tree of assets
+
+    Used for exposing asset structure for external tools.
+    """
+    query_projection = {
+        "_id": 1,
+        "data.tasks": 1,
+        "data.visualParent": 1,
+        "data.entityType": 1,
```

Comment:
Key `entityType` is ftrack specific. It should maybe not be considered as requirement.